onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorElementWise.cpp

```diff
@@ -560,13 +576,97 @@
         std::vector<DML_TENSOR_DESC> inputDescs = GetDmlInputDescs();
         std::vector<DML_TENSOR_DESC> outputDescs = GetDmlOutputDescs();
 
-        TOperatorDesc opDesc = {};
-        opDesc.InputTensor = &inputDescs[0];
-        opDesc.ScaleTensor = &inputDescs[1];
-        opDesc.ZeroPointTensor = &inputDescs[2];
-        opDesc.OutputTensor = &outputDescs[0];
+        if (hasZeroPointTensor)
+        {
+            TOperatorDesc opDesc = {};
+            opDesc.InputTensor = &inputDescs[OnnxInputIndex::inputIndex];
+            opDesc.ScaleTensor = &inputDescs[OnnxInputIndex::scaleIndex];
+            opDesc.ZeroPointTensor = &inputDescs[OnnxInputIndex::zeroPointIndex];
+            opDesc.OutputTensor = &outputDescs[0];
 
-        SetDmlOperatorDesc({ApiTraits::OperatorDescTraits<TOperatorDesc>::Type, &opDesc}, kernelInfo);
+            SetDmlOperatorDesc({ApiTraits::OperatorDescTraits<TOperatorDesc>::Type, &opDesc}, kernelInfo);
+        }
+        // Create a zero point tensor, since it's a required input by DML
+        else
+        {
+            auto inputSizes = m_inputTensorDescs[0].GetSizes();
+            TensorDesc intermediateOutputTensorDesc = TensorDesc(
+                    GetMlDataTypeFromDmlDataType(inputDataType),
+                    inputSizes,
+                    inputSizes,
+                    TensorAxis::DoNotCoerce,
+                    TensorAxis::W,
+                    TensorAxis::RightAligned,
+                    NchwDimensionCount, // minDimensionCount
+                    0 // guaranteedBaseOffsetAlignment
+                );
+            DML_TENSOR_DESC namedIntermediateOutputTensorDesc = intermediateOutputTensorDesc.GetDmlDesc();
+
+            // Create a tensor full of zeros
+            DML_FILL_VALUE_CONSTANT_OPERATOR_DESC zerosDesc = {};
+            zerosDesc.ValueDataType = inputDataType;
+            zerosDesc.OutputTensor = &namedIntermediateOutputTensorDesc;
+            const DML_OPERATOR_DESC opDesc1 = { DML_OPERATOR_FILL_VALUE_CONSTANT, &zerosDesc };
+
+            TOperatorDesc qLinearDesc = {};
+            qLinearDesc.InputTensor = &inputDescs[OnnxInputIndex::inputIndex];
+            qLinearDesc.ScaleTensor = &inputDescs[OnnxInputIndex::scaleIndex];
+            qLinearDesc.ZeroPointTensor = zerosDesc.OutputTensor;
+            qLinearDesc.OutputTensor = &outputDescs[0];
+            const DML_OPERATOR_DESC opDesc2 = { ApiTraits::OperatorDescTraits<TOperatorDesc>::Type, &qLinearDesc};
+
+            // Construct the graph
+            DML_INPUT_GRAPH_EDGE_DESC inputEdges[2];
```

Comment:
>DML_INPUT_GRAPH_EDGE_DESC inputEdges[2]; [](http://example.com/codeflow?start=12&length=40)

Would initialize to {} rather than leaving as randomness.

`DML_INPUT_GRAPH_EDGE_DESC inputEdges[2] = {}`